src/ray/gcs/gcs_server/gcs_placement_group_manager.cc

```diff
@@ -288,9 +288,10 @@ void GcsPlacementGroupManager::OnPlacementGroupCreationSuccess(
       absl::Microseconds(1);
   stats->set_scheduling_latency_us(scheduling_latency_us);
   stats->set_end_to_end_creation_latency_us(creation_latency_us);
-  ray::stats::STATS_placement_group_scheduling_latency_ms.Record(scheduling_latency_us /
-                                                                 1e3);
-  ray::stats::STATS_placement_group_creation_latency_ms.Record(creation_latency_us / 1e3);
+  ray::stats::STATS_gcs_placement_group_scheduling_latency_ms.Record(
```

Comment:
Do we get floating-point precision for ms stats?

Reply:
```
  RAY_LOG(ERROR) << "Sang creation latency " << creation_latency_us;
  RAY_LOG(ERROR) << "Sang creation latency divide by 1000 " << creation_latency_us / 1e3;
  RAY_LOG(ERROR) << "Sang creation latency passed to metrics " << (double) (creation_latency_us / 1e3);
```

```
(pid=gcs_server) [2021-12-13 20:17:43,546 E 83079 6684342] gcs_placement_group_manager.cc:290: Sang creation latency 1141
(pid=gcs_server) [2021-12-13 20:17:43,546 E 83079 6684342] gcs_placement_group_manager.cc:291: Sang creation latency divide by 1000 1.141
(pid=gcs_server) [2021-12-13 20:17:43,546 E 83079 6684342] gcs_placement_group_manager.cc:292: Sang creation latency passed to metrics 1.141
```

Based on this I think it does? Lmk if I misunderstood your question.